src/tests/integration/rest/rest.spec.ts

```diff
@@ -0,0 +1,146 @@
+import 'mocha';
+import * as Http from 'http';
```

Comment:
instead of importing whole modules, please prefer only importing the used symbols. In this case, there is a name clash, but that can be resolved by
```
import { createServer as createHTTPServer } from 'http';
import { createServer as createHTTPSServer, ServerOptions } from 'https';
```
For `fs`, only `readFileSync` seems to be needed and for `path` only `join` is needed.